src/pages/[platform]/build-a-backend/functions/configure-functions/index.mdx

```diff
@@ -0,0 +1,92 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Configure Functions',
+  description:
+    'Learn how to configure functions',
+    route: '/gen2/build-a-backend/functions/configure-functions',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+`defineFunction` comes out-of-the-box with sensible but minimal defaults. The following options are provided to tweak the function configuration.
+
+### `name`
+
+By default, functions are named based on the directory the `defineFunction` call is placed in. In the above example, defining the function in `amplify/functions/my-demo-function/resource.ts` will cause the function to be named `my-demo-function` by default.
+
+If an entry is specified, then the name defaults to the basename of the entry path. For example, an `entry` of `./signup-trigger-handler.ts` would cause the function name to default to `signup-trigger-handler`.
+
+This optional property can be used to explicitly set the name of the function.
+
+```ts title="amplify/functions/my-demo-function/resource.ts"
+export const myDemoFunction = defineFunction({
+  entry: './demo-function-handler.ts',
+  name: 'overrideName' // explicitly set the name to override the default naming behavior
+});
+```
+
+### `timeoutSeconds`
+
+By default, functions will time out after 3 seconds. This can be configured to any whole number of seconds up to 15 minutes.
+
+```ts title="amplify/functions/my-demo-function/resource.ts"
+export const myDemoFunction = defineFunction({
+  // highlight-next-line
+  timeoutSeconds: 60 // 1 minute timeout
+});
+```
+
+### `memoryMB`
+
+By default, functions have 512 MB of memory allocated to them. This can be configured from 128 MB up to 10240 MB. Note that this can increase the cost of executing the function. For more pricing information see [here](https://aws.amazon.com/lambda/pricing/).
+
+```ts title="amplify/functions/my-demo-function/resource.ts"
+export const myDemoFunction = defineFunction({
+  // highlight-next-line
+  memoryMB: 256 // allocate 256 MB of memory to the function.
+});
+```
+
+### `runtime`
+
+Currently, only Node runtimes are supported by `defineFunction`. However, you can change the Node version that is used by the function. The default is the oldest Node LTS version that is supported by AWS Lambda (currently Node 18).
```

Comment:
> (currently Node 18)

while this is informative, we will need to keep changing this on each change on the Lambda service. I think it may be better to link them to the Lambda documentation on this